### DISCUSSION TITLE
what is the Path Of This File on the computer,Gustavosta'sMagicPrompt model？

### DISCUSSION THREAD

**Discussion by zhai-hello** [adieyal/sd-dynamic-prompts#756] [Q&A]
help，I am a beginner.
I've already downloaded  all of the Gustavosta'MagicPrompt model to my computer from hugging face.
but,I don't know,what is the Path Of This File on the computer,Gustavosta'sMagicPrompt model？
please,and thank you!
![Snipaste_2024-04-08_15-10-20](https://github.com/adieyal/sd-dynamic-prompts/assets/150640855/207a4ba8-f6c1-4b23-80a4-0bdbc46446d9)

**Comment by rubenmejiac** [comment]
In WebUI-Forge, the models are located under system/transformers-cache. There are directories for each downloaded model, and another .locks folder that has related files too. I you don't need a model anymore, you can safely delete from there, and if you need it later just select it again within the UI with the model selector for Magic Prompts and it will download it to the mentioned locations.
